mis/solver/solver.py

```diff
@@ -235,7 +223,11 @@ def execute(self, pulse: Pulse, embedding: Register) -> Execution[Counter[str]]:
         Returns:
             Result: The solution from execution.
         """
-        return self.executor.submit_job(pulse, embedding)
+        assert self.backend is not None
```

Comment:
Should we not add an error message?

Personally I prefer to raise exceptions rather than assert things. e.g.
```python
if self.backend is not None:
     raise ValueError("Backend not set")
```
but I guess that `asserts` have already been used throughout, so it's ok.